README.md

```diff
@@ -116,6 +116,16 @@ With the CodePush plugin installed, configure your app to use it via the followi
  
 You are now ready to use the plugin in the application code. See the [sample applications](/samples) for examples and the API documentation for more details.
 
+## iOS specific options
```

Comment:
I think that this info should be as part of the previous section. Not the different section. 
You can add this info like `Note`. For example https://github.com/microsoft/cordova-plugin-code-push/pull/588/files#diff-04c6e90faac2675aa89e2176d2eec7d8R155

Reply:
Changed as requested. Thanks for the review.